docs/contributing.rst

```diff
@@ -80,8 +80,8 @@ guidelines when reporting a new bug.
 Writing patches
 ===============
 
-The better written a patch is, the higher chance that it'll get accepted and
-the sooner that will be merged.
+The better written a patch is, higher is the chance that it'll get accepted and 
```

Comment:
This line wasn't, and isn't, very idiomatic English. I would rephrase as "The better a patch is written, the higher the chances that it'll get accepted, and the sooner it will be merged".